rpc/legacyrpc/methods.go

```diff
@@ -1115,21 +1057,16 @@ func getNewAddress(icmd interface{}, w *wallet.Wallet) (interface{}, error) {
 		toReturn := make(map[string]string)
 		toReturn["address"] = addr.EncodeAddress()
 
-		ainfo, err := w.Manager.Address(addr)
+		pubKey, err := w.PubKeyForAddress(addr)
```

Comment:
Either can work.  But if this code isn't wrong in some way, I dont' see the point in changing it.